pkg/flowkit/services/accounts.go

```diff
@@ -281,9 +283,39 @@ func (a *Accounts) AddContract(
 	account *flowkit.Account,
 	contractName string,
 	contractSource []byte,
-	updateExisting bool,
 	contractArgs []cadence.Value,
+	contractFilename string,
```

Comment:
Would it make sense for these 4 arguments to be a `Contract` struct?

The function signature is a bit large now -- this could make it easier to call IMO.